communication/nats_dialog/dialog_establisher.go

```diff
@@ -2,7 +2,7 @@ package nats_dialog
 
 import (
 	"fmt"
-	"github.com/mgutz/logxi/v1"
```

Comment:
Why did we have `logxi` in the first place?
Maybe it makes sense to remove it from `glide.lock? It will safe us in the future from using logger which is not working.